### PR TITLE
Add support for in-app products

### LINKED ISSUE #181: Add support for in-app products
We'd like to automate publishing localized in-app products.

API: https://developers.google.com/android-publisher/api-ref/inappproducts

### PR DESCRIPTION
**Pull request by SUPERCILEX**
Fixes #181

### CODE REVIEW

**Comment by gtcompscientist** [comment]
Since this is built on top of another PR, I can't tell easily. Did you also add bootstrapping those?

**Comment by SUPERCILEX** [comment]
Yup:
https://github.com/Triple-T/gradle-play-publisher/blob/386f09bc9477f5d4cf438807d65b1aaad98d621a/plugin/src/main/kotlin/com/github/triplet/gradle/play/BootstrapTask.kt#L115-L120

File: plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherPlugin.kt [hunk 1]
```diff
@@ -120,6 +127,17 @@ class PlayPublisherPlugin : Plugin<Project> {
                 doFirst { logger.warn("$name is deprecated, use ${publishListingTask.get().name} instead") }
             }
 
+            val publishProductsTask = project.newTask<PublishProducts>(
+                    "publish${variantName}Products",
+                    "Uploads all Play Store in-app products for $variantName."
+            ) {
+                init()
+                productsDir = File(playResourcesTask.get().resDir, PRODUCTS_PATH)
```
Comment (bhurling) [inline plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherPlugin.kt]: Let's move this into a lazy block inside the task as in #354

Comment (SUPERCILEX) [reply plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherPlugin.kt]: 👍

**Pull request merged**